packages/os/link-kernel-modules.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Load additional kernel modules
+RefuseManualStart=true
+RefuseManualStop=true
+After=preconfigured.target
+Wants=preconfigured.target
```

Comment:
I'm not thrilled about this since we're sort of creating a limbo between `preconfigured.target` and `configured.target` where stuff happens but it's not in either phase.

I'd rather depend specifically on the services in `preconfigured.target` that we need to run after, and turn this into a required service for that target.